django/utils/translation/trans_real.py

```diff
@@ -529,7 +529,9 @@ def blankout(src, char):
 
 
 context_re = re.compile(r"""^\s+.*context\s+((?:"[^"]*?")|(?:'[^']*?'))\s*""")
-inline_re = re.compile(r"""^\s*trans\s+((?:"[^"]*?")|(?:'[^']*?'))(\s+.*context\s+((?:"[^"]*?")|(?:'[^']*?')))?\s*""")
+inline_re = re.compile(r"""^\s*trans\s+((?:"[^"]*?")|(?:'[^']*?'))"""
```

Comment:
Even by removing two tabs, the regex doesn't fit on 2 lines. Besides, the current split match the 3 parts of the regex: `trans`, the optional filters, and the optional context.
Should I resplit the regex anyway? Ignore the line length (the next regex already overflows)?